php/EE/Runner.php

```diff
@@ -47,19 +47,24 @@ public function __get( $key ) {
 	}
 
 	/**
-	 * Function to check and create the root directory for ee4.
+	 * Function to check and create the root directory for ee.
 	 */
-	private function init_ee4() {
+	private function init_ee() {
 
 		if ( ! is_dir( EE_CONF_ROOT ) ) {
-			mkdir( EE_CONF_ROOT );
+			$user = getenv('USER');
+			shell_exec('sudo mkdir -p ' . EE_CONF_ROOT);
+			shell_exec("sudo chown -R $user: " . EE_CONF_ROOT);
```

Comment:
Please remove this `chown` as we'll be tackling multi user usage of EasyEngine in later release.